CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 - Remove deprecated methods `Config::batch_report_interval` and `Config::batch_report_max_spans`.
 - Deprecate `full_name!()` and rename it to `full_path!()`.
 - Pass `Vec<SpanRecord>` to `Reporter::report()` instead of `&[SpanRecord]`.
+- Removed `SpanContext::encode_w3c_traceparent_with_sampled()`.
```

Comment:
`encode_w3c_traceparent_with_sampled` this is covered by the new `sampled` method?

A description or migration guide can be helpful here.